kubeflow-pipelines-visualization-server.yaml

```diff
@@ -30,13 +31,20 @@ pipeline:
     with:
       repository: https://github.com/kubeflow/pipelines
       tag: ${{package.version}}
-      expected-commit: 4467df5f9c8242d389f20214f75fecfc4119e8cd
+      expected-commit: 24ce44baf0070f76dcaf25d91a8c111de065c73f
 
   - uses: patch
     with:
       patches: 0001-Bump-dependencies.patch
 
   - runs: |
+      # When the requirements.in file needs updating, uncommenting these lines
+      # will install pip-compile, run it once to fetch packages, then fail the
+      # build to drop you into a debug shell:
+      # apk add py${{vars.py-version}}-pip-tools-bin
+      # pip-compile --output-file tmp.txt backend/src/apiserver/visualization/requirements.in
+      # false
```

Comment:
I think it will be better if we have this context in the patch file. instead of here. https://eng.inky.wtf/docs/squads/sustaining/playbooks/patch-creation-guide/

Reply:
I've moved it over there, thanks for catching this!